src/index.ts

```diff
@@ -223,6 +333,117 @@ async function getPortalItemInfo(
     return portalInfo;
 }
 
+/**
+ * Gets the set of parameters required for a given print template.
+ * @param itemId The portal item ID of the report item.
+ * @param portalUrl The URL of the ArcGIS Portal instance to use. Defaults to ArcGIS Online: "https://www.arcgis.com".
+ * @param serviceUrl The url to the template service.
+ * @param runToken A token used to authenticate with the service.
+ */
+export async function getMetadata(
+    itemId: string,
+    portalUrl: string,
+    serviceUrl: string,
+    runToken?: string
+): Promise<TemplateMetadata> {
+    // Infer the URL to the reporting service from the item
```

Comment:
I think we can remove this comment. It isn't true anymore.

Reply:
Done